scripts/nesi_scripts/install_fault.sl

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+# script version: slurm
+#
+# must be run with sbatch install_fault.sl [realisationDirectory] [OutputDirectory] [managementDBLocation]
+
+#SBATCH --job-name=install_fault
+#SBATCH --time=00:15:00
+#SBATCH --cpus-per-task=1
+
+if [[ -n ${CUR_ENV} && ${CUR_HPC} != "mahuika" ]]; then
+    source $CUR_ENV/workflow/install_workflow/helper_functions/activate_env.sh $CUR_ENV "mahuika"
+fi
+VM_PARAMS=$1
```

Comment:
i would suggest adding a quick check to make sure all arguments are not empty.
such as 
${1:?please provide the path to VM_PARAMS } 
for all of them, just in case there was a unexpected change in the script above.